src/styles/blocks/catalog.scss

```diff
@@ -18,8 +19,17 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: width, height, color, 300ms linear;
```

Comment:
Please use variable for transition time

Also, you may use transform instead of width, height
`transition: transform, color, 300ms linear;`